Add Hydrogen

benjaminsehl opened the pull request:
Adds:
```md
- [Hydrogen](https://github.com/shopify/awesome-hydrogen#readme) - Edge-first framework for building Shopify storefronts with React
```

<!-- Congrats on creating an Awesome list! 🎉 -->

<!-- Please fill in the below placeholders -->

**[Insert URL to the list here]**
https://github.com/shopify/awesome-hydrogen#readme

**[Explain what this list is about and why it should be included here]**
This list is to help users of the Hydrogen framework find relevant tools and resources to help them accelerate their build. It is a place for us to show off the best of the community and various integrations with the Hydrogen framework.

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration. **Just commenting “looks good” or simply marking the pull request as approved does not count!** You have to actually point out mistakes or improvement suggestions.
- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/main/create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `add Swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The title of your entry should be title-cased and the URL to your list should end in `#readme`.
	- Example: `- [Software Architecture](https://github.com/simskij/awesome-software-architecture#readme) - The discipline of designing and building software.`
- The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it. **Instead use [#2242](https://github.com/sindresorhus/awesome/issues/2242) for incubation visibility**.
- Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- The default branch should be named [`main`, not `master`](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/).
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- Non-generated Markdown file in a GitHub repo.
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate. Please search for existing submissions.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/main/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
	- Must not feature `Contributing` or `Footnotes` sections.
- Has an appropriate license.
	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=main&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and [Unlicense](https://unlicense.org).
	- Place a file named `license` or `LICENSE` in the repo root with the license text.
	- **Do not** add the license name, text, or a `Licence` section to the readme. GitHub already shows the license name and link to the full text at the top of the repo.
	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/main/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
	- It can optionally be linked from the readme in a dedicated section titled `Contributing`, positioned at the top or bottom of the main content.
	- The section should not appear in the Table of Contents.
- All non-important but necessary content (like extra copyright notices, hyperlinks to sources, pointers to expansive content, etc) should be grouped in a `Footnotes` section at the bottom of the readme. The section should not be present in the Table of Contents.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't use [hard-wrapping](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap).
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

prokopsimek commented:
@benjaminsehl You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).

```
README.md:46:203
  ✖   1:1    Missing file contributing.md                                  remark-lint:awesome-contributing
  ✖   1:1    The repository should have a description                      remark-lint:awesome-github
  ✖   1:1    The repository should have "awesome" as a GitHub topic        remark-lint:awesome-github
  ✖   1:1    The repository should have "awesome-list" as a GitHub topic   remark-lint:awesome-github
  ✖   1:1    License was not detected by GitHub                            remark-lint:awesome-github
  ✖   1:1    Missing or invalid Table of Contents                          remark-lint:awesome-toc
  ✖  21:120  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  46:199  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  46:203  https://github.com/igrigorik                                  remark-lint:double-link
  ✖  47:146  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  48:156  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  48:160  https://github.com/jplhomer                                   remark-lint:double-link
  ✖  49:135  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  53:[11](https://github.com/sindresorhus/awesome/runs/6991949330?check_suite_focus=true#step:3:12)2  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  53:1[18](https://github.com/sindresorhus/awesome/runs/6991949330?check_suite_focus=true#step:3:19)  https://github.com/igrigorik                                  remark-lint:double-link
  ✖  54:89   List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  54:95   https://github.com/jplhomer                                   remark-lint:double-link
  ✖  55:82   List item link and description must be separated with a dash  remark-lint:awesome-list-item
```

sindresorhus commented:
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

benjaminsehl commented:
Thanks for the feedback @sindresorhus / @prokopsimek — I've reviewed two other open PR's with feedback on which pieces they were missing. We also updated our Awesome List to include the contributing and license files,  and we fixed our formatting errors.

benjaminsehl commented:
I have reviewed #2099 (as referenced in #2320) & #2169

@sindresorhus — while the author of #2320 went with my suggestion, you asked not to do that. I will continue to monitor the pull requests to add proper suggestions in case this does not count as two different reviews.

Please let me know if there is anything else I've missed on the above — I believe I've now followed all of the guidelines correctly.

benjaminsehl commented:
`unicorn`

lockedwimp45 commented:
I have no idea what this is please stop sending it to me I'll unsubscribe

On Mon, Jul 11, 2022, 9:19 AM Benjamin Sehl ***@***.***>
wrote:

> I have reviewed #2099 <https://github.com/sindresorhus/awesome/pull/2099>
> (as referenced in #2320
> <https://github.com/sindresorhus/awesome/pull/2320>) & #2169
> <https://github.com/sindresorhus/awesome/pull/2169>
>
> @sindresorhus <https://github.com/sindresorhus> — while the author of
> #2320 <https://github.com/sindresorhus/awesome/pull/2320> went with my
> suggestion, you asked not to do that. I will continue to monitor the pull
> requests to add proper suggestions in case this does not count as two
> different reviews.
>
> Please let me know if there is anything else I've missed on the above — I
> believe I've now followed all of the guidelines correctly.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/sindresorhus/awesome/pull/2289#issuecomment-1180536305>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZYA2L6OVPUWNBXYQZTGBP3VTQ3R5ANCNFSM5ZNRUV5A>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

justlevine commented:
My first time reviewing, but based on the PR checklist this list looks`awesome` to me.

The list has been open for 30+ days, it's got the proper tags, badge, and contributing guidelines, and includes a TOC and the item summaries are succinct yet clear.

And, while not one of the criteria, I personally found the list helpful.

benjaminsehl commented:
unicorn

benjaminsehl commented:
@sindresorhus Just curious what I'm missing here. Happy to do whatever you need to get this merged!

sindresorhus commented:
Tweet: https://twitter.com/awesome__re/status/1594064715247849473
